phpcr-shell/installation.rst

```diff
@@ -43,7 +43,7 @@ Simply add the shell to your ``composer.json`` file
         ...
         require: {
             ...
-            "phpcr-shell": "<latest version here>"
+            "phpcr/phpcr-shell": "<latest version here>"
```

Comment:
i did `composer require phpcr/phpcr-shell` and it automatically resolved to `~1.0@dev`. shall we just propose composer instead?